pkg/thumbnail/imgsource/webdav.go

```diff
@@ -27,12 +28,17 @@ type WebDav struct {
 func (s WebDav) Get(ctx context.Context, file string) (image.Image, error) {
 	u, _ := url.Parse(s.baseURL)
 	u.Path = path.Join(u.Path, file)
+	fmt.Printf("url: %s", u.String())
```

Comment:
I'd use a logger, not print

Reply:
perhaps adding a logger to the type: https://github.com/owncloud/ocis-thumbnails/blob/use-ocis-proxy/pkg/thumbnail/imgsource/webdav.go#L23-L25

Reply:
I'll remove this, it was purely for debugging